### PR TITLE
Aug 2019 update

### PR DESCRIPTION
**Pull request by jennpb**
## Who is this for?
Collections users

## What is it doing for them?
Letting them know about recent changes to search

### CODE REVIEW

File: catalogue/webapp/pages/PROGRESS_NOTES.md [hunk 1]
```diff
@@ -7,32 +7,40 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
...
 
-## What’s new? (updated June 2019)
+## What’s new? (updated 16 August 2019)
 
-We made it easier to get an overview of a book's content and review individual pages in detail. We are currently working to improve the relevance of your search results which will be released shortly.
+### More of the collection available to search
+We've made more of our digital collections available under an Audio/Video filter. You can now find: 
+- 4337 multi-volume works
+- 900 digitised videos and 
+- 3 audio files [(including an 1890 recording of Florence Nightingale)](https://wellcomecollection.org/works/tp9njewm)
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/PROGRESS_NOTES.md]: ✨

Comment (gestchild) [reply catalogue/webapp/pages/PROGRESS_NOTES.md]: Does this imply that the multi-volume works are available under the Audio/Video filter?

Comment (jennpb) [reply catalogue/webapp/pages/PROGRESS_NOTES.md]: Ach good catch.

Comment (jennpb) [reply catalogue/webapp/pages/PROGRESS_NOTES.md]: fixed now.

**Pull request merged**